app/views/case_court_reports/index.html.erb

```diff
@@ -12,13 +12,15 @@
     <%= form_with url: generate_case_court_reports_path, local: false do |form| %>
       <div class="field form-group">
         <%= label_tag :case_number, t(".label.case_number") %>
-        <% select_options = @assigned_cases.map { |casa_case| casa_case.decorate.court_report_select_option } %>
-        <%= select_tag :case_number,
-                       options_for_select(select_options),
-                       prompt: t(".prompt.select_case_number"),
-                       include_blank: false,
-                       id: "case-selection",
-                       class: "custom-select" %>
+        <select id="case-selection" name="case_number" class="custom-select select2">
+          <option value="" selected><%= t(".prompt.select_case_number") %></option>
+          <% @assigned_cases.each do |casa_case| %>
+            <% option = casa_case.decorate.court_report_select_option %>
+            <option value="<%= option[1] %>" data-lookup="<%= casa_case.assigned_volunteers.map(&:display_name).join(",") %>">
```

Comment:
I had to abandon the use of the `select_tag` because I needed to add the `data-lookup` attribute. Is there a way to add this attribute but still use the `select_tag`?